Unix/tools/ktstrip

```diff
@@ -43,59 +43,28 @@ then
    exit -2
 fi
 
-TMP_FILE=$(mktemp)
-list=$(printf "read_kt $SRC_KT\nlist\n" | ktutil | awk '/[0-9]/ { print " "$1"|" $2"|"$3 }' )
+list=$(mktemp)
+delete_list=$(mktemp)
+cmd=$(mktemp)
+tmp_kt="/tmp/tmp.kt"
 
-declare -a remove_entries=()
-for princ in $list
-do
-    IFS='|' read -r -a principal <<< "$princ"
-#    echo "Principal " "${principal[2]}"
-    principalname="${principal[2]}"
-    entryno="${principal[0]}"
-    keyvers="${principal[1]}"
-    
-    case $principalname in
-       host/*) printf  $principalname"\n" ;;
-       RestrictedKrbHost/*) printf $principalname"\n" ;;
-       *) printf "remove " $principalname "\n" 
+printf "read_kt $SRC_KT\nlist\n" | ktutil >$list  
 
-          remove_entries+=($entryno)
-          ;;
-    esac
-done
+# sort -rV reverse sorts the entries by "version number" which is to say actual numeric value. -n sorts by digit.
```

Comment:
Would be really nice to have sample output and a description (in comment) of exactly what you're doing here.